request_a_govuk_domain/request/views.py

```diff
@@ -153,24 +162,26 @@ class RegistrantTypeView(FormView):
 
     def get(self, request):
         params = {}
-        if 'change' in request.GET:
-            params['registrant_type'] = request.session['registration_data']['registrant_type']
+        if "change" in request.GET:
+            params["registrant_type"] = request.session["registration_data"][
+                "registrant_type"
+            ]
             form = RegistrantTypeForm(params)
         else:
             form = RegistrantTypeForm()
-        return render(request, self.template_name, {'form': form})
+        return render(request, self.template_name, {"form": form})
 
     def post(self, request):
         form = RegistrantTypeForm(request.POST)
         if form.is_valid():
-            registration_data = request.session.get('registration_data', {})
-            registration_data['registrant_type'] = form.cleaned_data['registrant_type']
-            request.session['registration_data'] = registration_data
+            registration_data = request.session.get("registration_data", {})
+            registration_data["registrant_type"] = form.cleaned_data["registrant_type"]
+            request.session["registration_data"] = registration_data
             if form.cleaned_data["registrant_type"] == "none":
-                return redirect('registrant_type_fail')
+                return redirect("registrant_type_fail")
             else:
-                return redirect('confirm')
```

Comment:
Now that we have the Registrant page in place, we should call that page rather than confirm page here.